### PR TITLE
Update manuscript and references

### PR DESCRIPTION
**Pull request by namn12**

### CODE REVIEW

File: manuscript/figure_captions/figure_1_caption.md [hunk 1]
```diff
@@ -0,0 +1 @@
+![** tHMM influence diagram for $𝑵$ cells with $𝑲$ states. The latent states ($\bm{z}$) represent subpopulation classification of each cell. The observed states ($\bm{x}$) in blue/orange denote measured phenotypes, particularly fitness (i.e. end-of-life fate by binary outcome variable $\phi$, and lifetime by variable $\tau$ in hours). The tHMM is robust to cells that die, such as cell $6$.. ...(./Figures/figure2.svg){#fig:tfac}
```
Comment (aarmey) [inline manuscript/figure_captions/figure_1_caption.md]: Figures should be included within the text approximately where they should appear.

Comment (aarmey) [reply manuscript/figure_captions/figure_1_caption.md]: Also, I don't think you ever close your `[` bracket?

File: manuscript/text/03_results.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+## Lineage Length Scaling
+
+Thus, the lineage was recreated 200 times with varying lengths, and the results of the accuracy prediction and distribution parameters $(\theta<sub>B</sub>, \lambda<sub>E</sub>)$ are seen in Figure 4. The accuracy of the model in assigning states improves with increased number of cells within a lineage. Additionally, the parameter estimation approaches that of the true value for each subpopulation. In particular, the model is more accurate in assigning correct parameter estimates for resistant cells. This is attributed to robustness of the tHMM in identifying cell transitions and performing maximum likelihood estimation to generate an accurate $\bm{E}$ matrix (Eq(3)).
```
Comment (aarmey) [inline manuscript/text/03_results.md]: Subscript in an equation is `_{B}`.

File: manuscript/text/01_introduction.md [hunk 3]
```diff
@@ -25,28 +25,44 @@ affiliation:
...
 
 link-citations: true
 
 csl: ./manuscript/templates/nature.csl
 
 ---
 
-# Summary points
+# Summary points -  this may or may not be neede depending on the journal we apply to
```
Comment (aarmey) [inline manuscript/text/01_introduction.md]: Good to put together regardless.

File: manuscript/text/01_introduction.md [hunk 4]
```diff
@@ -25,28 +25,44 @@ affiliation:
...
 
 - One
 - Two
 - Three
 
-# Author Summary
+# Author Summary - this may or may not be neede depending on the journal we apply to
```
Comment (aarmey) [inline manuscript/text/01_introduction.md]: Remove note.

File: manuscript/text/01_introduction.md [hunk 5]
```diff
@@ -25,28 +25,44 @@ affiliation:
...
...atments of cancer consists of chemotherapy, particularly targeted therapies, whereby patients are given drugs that destroy highly-prolific cells to stall cancer growth or eliminate the tumor. However, long-term therapeutic efficacy varies significantly due to the vast heterogeneity in intratumor response to therapy.[@Di Maio; @De Roock] Specifically, each tumor consists of cancer cell subpopulations that differ in terms of cell intrinsic and extrinsic factors including genetic plasticity, epigenetic alterations, and micro-environmental stressors.[@Semenza; @Nagarsheth; @Feinberg; @Falkenberg] Current drug-screening protocols involve giving known cancer cell lines different drug doses and evaluating therapy performance based on the percent of cells eliminated and the dose required for half-maximal effect.[@Chauvin; @Zhang_L] These metrics, however, provide population averages of overall tumor response to the therapy and fail to consider the subpopulation heterogeneity. In addition, there is a need for real-time cell classification, which would identify the significant proportion of cells that undergo stochastic changes in resistance after onset of therapy.[@Gupta; @Enderling; @Inde]
```
Comment (aarmey) [inline manuscript/text/01_introduction.md]: Are you sure spaces can be in the cite-key?

File: manuscript/text/01_introduction.md [hunk 6]
```diff
@@ -25,28 +25,44 @@ affiliation:
...
+
+The phenotypic measurements and tHMM extrapolation are able to be conducted in real-time and do not require extensive laboratory resources characteristic of ‘omics’ techniques. Identification of cell resistance can be achieved relatively early within the growth of a cell mass or onset of a therapy, depending on the growth characteristics of the cell lineages, which are explored in Section 6. The tHMM pipeline in its current version is provided as an open source toolkit in Python (https://github.com/meyer-lab/lineage-growth), enabling other investigators to understand the identities of heterogeneous subpopulations in both basic research settings and clinical analysis. This computational tool is equipped with class modules and functions that can be easily edited to serve a user’s specific purpose but can also be used immediately. We also provide tools for lineage data preprocessing, lineage data input and output, and lineage data visualization, apart from the main methods involving tHMM analysis.
```
Comment (aarmey) [inline manuscript/text/01_introduction.md]: I think it's triangular brackets `<>` for a URL you want to display...

**Comment by shak360** [comment]
pandoc-citeproc: reference Di not found
pandoc-citeproc: reference De not found
pandoc-citeproc: reference De not found
pandoc-citeproc: reference Huang not found
pandoc-citeproc: reference O not found
pandoc-citeproc: reference Huang not found

might be the reason for failing? Issue with Bibtex ids.

File: manuscript/text/01_introduction.md (hunk 5 above)
Comment (shak360) [reply manuscript/text/01_introduction.md]: No they can't, pretty sure they're the reason for failure.

**Comment by shak360** [comment]
Build issues with citations are fixed. Is now an issue with latex somewhere...

File: manuscript/text/01_introduction.md (hunk 5 above)
Comment (shak360) [reply manuscript/text/01_introduction.md]: Fixed.

File: manuscript/figure_captions/figure_1_caption.md (hunk 1 above)
Comment (namn12) [reply manuscript/figure_captions/figure_1_caption.md]: ![**Overall title. ** A) blah.  B) blah blah ...(./Figures/figure3.svg){#fig:tfac}

This is the example format i was told to use ....

Can you give me the way to input figure captions? Where do I close the bracket? After title or whole text

Comment (namn12) [reply manuscript/figure_captions/figure_1_caption.md]: i put it after the title. still failes

File: manuscript/text/03_results.md (hunk 2 above)
Comment (namn12) [reply manuscript/text/03_results.md]: done

File: manuscript/text/01_introduction.md (hunk 6 above)
Comment (namn12) [reply manuscript/text/01_introduction.md]: done

**Comment by namn12** [comment]
Does this mean we inputted the references incorrectly that it sees no citations? Where is the manuscript log file?

Biber warning: [86] Utils.pm:193> WARN - The file 'manuscript.bcf' does not contain any citations!
This is XeTeX, Version 3.14159265-2.6-0.99999 (TeX Live 2019/dev/Debian) (preloaded format=xelatex)
 restricted \write18 enabled.
entering extended mode
Latexmk: Run number 1 of rule 'xdvipdfmx'
Biber warning: [86] Utils.pm:193> WARN - The file 'manuscript.bcf' does not contain any citations!
For rule 'xdvipdfmx', running '&do_viewfile(  )' ...
Collected error summary (may duplicate other messages):
  xelatex: Command for 'xelatex' gave return code 1
      Refer to 'manuscript.log' for details
make: *** [makefile:21: manuscript/manuscript.pdf] Error 12

**Comment by shak360** [comment]
No it's an issue with the latex somewhere not this citation somewhere. I should be able to find it tomorrow morning.

**Comment by namn12** [comment]
> No it's an issue with the latex somewhere not this citation somewhere. I should be able to find it tomorrow morning.

how can u tell from that error? just so i can identify what to do next time this shows up. thanks

**Comment by shak360** [comment]
It's just happened in the past where this error has shown up and it happened to be a \LaTeX syntax error. I might be wrong in interpreting the error log, but a similar error in the past was fixed by adding the appropriate package in the latex instruction file.

**Comment by aarmey** [comment]
The cite keys with spaces will definitely cause an error. Fix those and if it still isn't clear what's failing, ping me.

**Comment by shak360** [comment]
Got it! There was an extra { ... Still need help setting up the `align` environment for the equations depicted in 
image
@aarmey @namn12 But should be easier to read since it builds as a PDF now.

**Comment by aarmey** [comment]
Oh, I think this kind of align environment can just all go between the dollar signs. It's just one big equation.

File: manuscript/text/03_results.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# Results
+
+![** tHMM influence diagram for $\bm{N}$ cells with $\bm{K}$ states.** The latent states ($\bm{z}$) represent subpopulation classification of each cell. The observed states ($\bm{x}$) in blue/orange denote measured phenotypes, particularly fitness (i.e. end-of-life fate by binary outcome variable $\phi$, and lifetime by variable $\tau$ in hours). The tHMM is robust to cells that die, such as cell $6$.. ...](./figures/figure2.svg){#fig:tfac}
```
Comment (aarmey) [inline manuscript/text/03_results.md]: (1) Figure captions should be inline with text. (2) Note the position of the `]` here—brackets should encompass all of the caption. (3) Just topic phrase should be bold (enclosed in `**`). (3) Note `figure` folder is lower case.

Comment (namn12) [reply manuscript/text/03_results.md]: great thank you.
done
@shak360 waiting for your approval

Comment (shak360) [reply manuscript/text/03_results.md]: Will do Thursday morning, if you can wait. Need to add the first `align` equation that I took out. There is an error with the citation early on in the introduction. We also have the error : `Latex found 4 multiply defined reference(s)` which is causing the build to fail. Can also add the table I think. The bolding in the figure captions is also not working.

Comment (aarmey) [reply manuscript/text/03_results.md]: `fig:tfac` is the reference name used for this figure. Those need to be unique for each figure.

**Comment by shak360** [comment]
@namn12 "In 2018, over 1.7 million new cases of cancer were estimated to be diagnosed with over 609,000 of those cases projected to be fatal.[@Smith; @] " in the introductions paragraph need to be changed. I'm talking about the empty citation.

**Comment by shak360** [comment]
finally building... Some of the figures are still blank though? I'm assuming it's because the actual svg files never get updated? @namn12 should be easier for you to work with again.

**Pull request merged**